README.md

```diff
@@ -37,7 +37,7 @@ The fields in the table below can be used in these parts of STAC documents:
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| proj:code        | string\|null   | Authority and specific code of the data source (e.g., ["EPSG:####"](https://epsg.org/), ["IAU:#####"](http://www.opengis.net/def/crs/IAU/2015)) |
+| proj:srid        | string\|null   | Authority and specific code of the data source (e.g., ["EPSG:####"](https://epsg.org/), ["IAU:#####"](http://www.opengis.net/def/crs/IAU/2015)) |
```

Comment:
I really don't like srid. It's not very intuituve. Isn't code or id more reasonable?

Reply:
Some like code, come like SRID. I do not know that we a consensus here. Here are some [ESRI docs](https://desktop.arcgis.com/en/arcmap/latest/manage-data/using-sql-with-gdbs/what-is-an-srid.htm#:~:text=A%20spatial%20reference%20identifier%20(SRID,use%20to%20store%20your%20data.) describing that this is the SRID.